### ISSUE TITLE
Error during cloudml_train() with R 3.6

### ISSUE THREAD

**Issue by pmags** [rstudio/cloudml#190]
Recently, everytime i use cloudml_train(scrip.R) i receive the following error mistake:

Command ['Rscript', '/root/.local/lib/python2.7/site-packages/cloudml-model/cloudml/deploy.R'] failed: exit code 1

I have try to train two simple example scripts as below:

`
 dir.create("mnist-train")
 file.copy(system.file("examples/mnist/train.R", package = "cloudml"), "mnist-train")
 setwd("mnist-train")
 cloudml::cloudml_train()
`

`
library(tfestimators)
load("mtcars.rda")

mtcars_input_fn <- function(data, num_epochs = 1) {
  input_fn(data, 
           features = c("disp", "cyl"), 
           response = "mpg",
           batch_size = 32,
           num_epochs = num_epochs)
}

mtcars_input_fn_caret <- function(data, num_epochs = 1){
  input_fn(mpg ~ disp + cyl,
           data = data,
           batch_size = 32,
           num_epochs = num_epochs)
}

cols <- feature_columns( 
  tf$feature_column$numeric_column("disp"),
  tf$feature_column$numeric_column("cyl")
)

model <- linear_regressor(feature_columns = cols)

set.seed(40)
indices <- sample(1:nrow(mtcars), size = 0.80 * nrow(mtcars))
train <- mtcars[indices, ]
test  <- mtcars[-indices, ]

model %>% train(mtcars_input_fn(train, num_epochs = 10))
model %>% evaluate(mtcars_input_fn(test))
obs <- mtcars[1:3, ]
model %>% predict(mtcars_input_fn(obs))
`

They both run locally without any problem so i don't believe and given the examples i don't believe it is a train script issue. 

This started recently and the only change this past month was an update to R 3.6
I reinstalled cloudml and the problem persist and i tried running the same scripts inside a Rprojet through Rstudio and outside any Rproject as a stand alone script and everytime the run returns the same mistake.

Might it be a compatibility issue with the new R version?

My system info is as follows:
`
─ Session info ─────────────────────────
 setting  value                       
 version  R version 3.6.0 (2019-04-26)
 os       Ubuntu 18.04.2 LTS          
 system   x86_64, linux-gnu           
 ui       RStudio                     
 language (EN)                        
 collate  en_US.UTF-8                 
 ctype    pt_PT.UTF-8                 
 tz       Europe/Lisbon               
 date     2019-05-12                  

─ Packages ─────────────────────────────
 package     * version date       lib
 assertthat    0.2.1   2019-03-21 [1]
 backports     1.1.3   2018-12-14 [2]
 base64enc     0.1-3   2015-07-28 [2]
 callr         3.1.1   2018-12-21 [2]
 cli           1.1.0   2019-03-19 [1]
 cloudml     * 0.6.0   2018-08-27 [1]
 config        0.3     2018-03-27 [1]
 crayon        1.3.4   2017-09-16 [2]
 desc          1.2.0   2018-05-01 [1]
 devtools      2.0.2   2019-04-08 [1]
 digest        0.6.18  2018-10-10 [2]
 fs            1.2.6   2018-08-23 [2]
 glue          1.3.1   2019-03-12 [1]
 jsonlite      1.6     2018-12-07 [2]
 lattice       0.20-38 2018-11-04 [4]
 magrittr      1.5     2014-11-22 [2]
 Matrix        1.2-17  2019-03-22 [4]
 memoise       1.1.0   2017-04-21 [1]
 packrat       0.5.0   2018-11-14 [1]
 pkgbuild      1.0.3   2019-03-20 [1]
 pkgload       1.0.2   2018-10-29 [1]
 prettyunits   1.0.2   2015-07-13 [2]
 processx      3.2.1   2018-12-05 [2]
 ps            1.3.0   2018-12-21 [2]
 R6            2.4.0   2019-02-14 [1]
 Rcpp          1.0.1   2019-03-17 [1]
 remotes       2.0.4   2019-04-10 [1]
 rlang         0.3.4   2019-04-07 [1]
 rprojroot     1.3-2   2018-01-03 [1]
 rstudioapi    0.8     2018-10-02 [2]
 sessioninfo   1.1.1   2018-11-05 [1]
 tfruns      * 1.4     2018-08-25 [1]
 usethis       1.5.0   2019-04-07 [1]
 whisker       0.3-2   2013-04-28 [2]
 withr         2.1.2   2018-03-15 [2]
 yaml          2.2.0   2018-07-25 [2]
`

**Comment by Sophia0616** [comment]
It appears so. 

From the R documentation from 3.6, the rds written by R 3.6 is not readable by previous versions. I was able to fix this:

`Sys.setenv(R_CONFIG_ACTIVE = "cloudml", 
          R_DEFAULT_SAVE_VERSION = 2, 
          R_DEFAULT_SERIALIZE_VERSION = 2
)`

placed it in deploy.R 

You can use my repo https://github.com/Sophia0616/cloudml or fork it and make the changes yourself.

I have no experience in writing cran libraries of any kind, so I hope someone with more experience than I do can tell me if I did something stupid. Always happy to learn. 

Hope that helps :)

**Comment by dfalbel** [comment]
This should be fixed in the Github version of cloudml
